app.rb

```diff
@@ -0,0 +1,39 @@
+require 'sinatra/base'
+require_relative './lib/user'
+require_relative './lib/peep'
+
+class Chitter < Sinatra::Base
+  enable :sessions
+  set :public_folder, Proc.new { File.join(root, 'public') }
+
+  get '/' do
+    @peeps = Peep.all
+    erb(:index)
+  end
+
+  get '/user/new' do
+    erb(:sign_up)
+  end
+
+  post '/user/new' do
+    session[:user] = User.create(params[:username])
+    redirect '/'
+  end
+
+  get '/peep/new' do
+    erb(:new_peep)
+  end
+
+  post '/peep/new' do
+    Peep.create(@current_user.id, params[:content], Time.now.strftime("%Y/%m/%d"), Time.now.strftime("%H:%M:%S"))
```

Comment:
I wonder if you could have the time arguments be default initialize parameters within the Peep class?